qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -1330,6 +1330,20 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                             vals=vals.Enum(*list(self._samplingrate_codes.keys()))
                             )
 
+        self.add_parameter('scope_samplingrate_float',
+                           label="Scope's sampling rate as float",
+                           set_cmd=partial(self._set_samplingrate_as_float,
+                                           self._samplingrate_codes),
+                           unit='Hz',
+                           get_cmd=partial(self._get_samplingrate_as_float),
+                           docstring=""" Set the scope sampling rate with float.
```

Comment:
i personally think the word "float" is abused in this PR. smth like "numeric" or "number" seems to be better. but i leave this to the author.

Reply:
✔️